examples/structured_data/movielens_recommendations_transformers.py

```diff
@@ -0,0 +1,606 @@
+"""
+Title: Recommendations with Transformer
+Author: [Khalid Salama](https://www.linkedin.com/in/khalid-salama-24403144/)
+Date created: 2020/12/30
+Last modified: 2020/12/30
+Description: Using the Behavior Sequence Transformer model for on Movielens dataset.
+"""
+
+"""
+## Introduction
+
+This example demonstrates the [Behavior Sequence Transformer (BST)](https://arxiv.org/abs/1905.06874)
```

Comment:
Suggestion: rewrite as: the example demonstrates _how to_ [build and train] the BST model _to predict the user click-through rate (it's in the paper: (https://arxiv.org/abs/1905.06874))
```suggestion
This example demonstrates how to build and train the [Behavior Sequence Transformer (BST)](https://arxiv.org/abs/1905.06874) model to predict the user click-through rate.
```